perf/lua/1mops_write.lua

```diff
@@ -28,12 +28,25 @@ local params = require('internal.argparse').parse(arg, {
 
 local test_dir = fio.tempdir()
 
+local function rmtree(s)
```

Comment:
Please add a check for root directory (`/`). It would be very-very bad if we will do `rm -rf /` on user's machine.

Reply:
It's a local function, never exposed to the user. The only use of the function is to remove the contents of the `tmp_dir` generated by the `fio.tempdir()`. Hence, no check is necessary.

Reply:
Right, thanks!